Update `CONTRIBUTING.md` on `pull` (#132)
SpineEventEngine/config: CONTRIBUTING.md

How to contribute
==================
Thank you for wanting to contribute to Spine. The following links will help you get started:
 * [Wiki home][wiki-home] — the home of the framework developer's documentation.
 * [Getting started with Spine in Java][quick-start] — this guide will walk you through
   a minimal client-server “Hello World!” application in Java. 
 * [Introduction][docs-intro] — this section of the Spine Documentation will help you understand
   the foundation of the framework.
   
Pull requests
-------------

> A work on an improvement starts from creating an issue which describes a bug or a feature you
> intend to fix. The issue will be used for communications on the proposed improvements. If code
> changes are going to be introduced, the issue would also have a link to corresponding Pull Request.

Reviewer: I'd just drop the `intend to fix` part, cause we do not `fix` features and if we add `implement` it'd be just more cluttered.

```suggestion
The work on an improvement starts with creating an issue that describes a bug or a feature. The issue will be used for communications on the proposed improvements. 
If code changes are going to be introduced, the issue should also have a link to the corresponding Pull Request.
```

Code contributions should:
  * Be accompanied by tests.

>   * Be licensed Apache 2.0 with the appropriate copyright header for each file.

Reviewer: Maybe worth adding a link to the license?
```suggestion
  * Be licensed under the Apache v2.0 license with the appropriate copyright header for each file.
```

  * Formatted according to the code style. See [Wiki home][wiki-home] for the links to
    style guides of the programming languages used in the framework.  

Contributor License Agreement
-----------------------------
Contributions to the code of Spine Event Engine framework and its libraries must be accompanied by

> Contributor License Agreement.  

Reviewer: We're using the CLA below, maybe worth pinpointing it here?
```suggestion
Contributor License Agreement (CLA).
```
